src/assets/src/components/queue.tsx

```diff
@@ -446,14 +446,27 @@ export function QueuePage(props: PageProps<QueuePageParams>) {
         await api.removeMeeting(queue!.my_meeting!.id);
     }
     const [doLeaveQueue, leaveQueueLoading, leaveQueueError] = usePromise(leaveQueue);
-    const confirmLeaveQueue = (queueStatus: 'open' | 'closed') => {
-        const loseMessage = 'By leaving the queue, you will lose your place in line'
-        const description = 'Are you sure you want to leave the queue? ' + (
-            queueStatus === 'open'
-                ? loseMessage + '.'
-                : `The queue is currently closed. ${loseMessage} and will not be able to rejoin until the queue is reopened.`
+    const confirmLeaveQueue = (queueStatus: 'open' | 'closed', meetingStatus: MeetingStatus | undefined) => {
```

Comment:
It might be nice to move this out of `queue.tsx`, but I wasn't really sure where else to put it.